sys/net/gnrc/network_layer/ipv6/nib/_nib-6ln.c

```diff
@@ -185,12 +185,19 @@ void _handle_rereg_address(const ipv6_addr_t *addr)
 {
     gnrc_netif_t *netif = gnrc_netif_get_by_ipv6_addr(addr);
 
+    if (netif == NULL) {
+        DEBUG("nib: Couldn't re-register %s, no current router found or address "
+              "wasn't assigned to any interface anymore.\n",
```

Comment:
I would keep the "no current router found" message in the `else` branch of the `router_reachable`-if. This is after all what this check does and with the current text it might cause confusion.
```suggestion
        DEBUG("nib: Couldn't re-register %s, address wasn't assigned to any "
              "interface anymore.\n",
```